### ISSUE TITLE
curvenote start fails - nodejs installed with mamba

### ISSUE THREAD

**Issue by johnjarmitage** [curvenote/curvenotejs#112]
So I tested `curvenote start` on both CentOS and Ubuntu, and both show the same error:
```
	✨✨✨  Starting Curvenote  ✨✨✨


📖 Built egu22/presentation in 363 ms.
📖 Built egu22/README in 1.58 s.
⚙️  Writing config.json
📚 Built 2 / 2 pages in 1.58 s.
The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
```
On both linux machines I install `nodejs` via mamba using the `environment.yml` within https://github.com/johnjarmitage/EGU22 in order to have `npm` to install and run `curvenote`. This is the only thing in common, otherwise the two machines are very different. I specify that `nodejs<17` and update `npm` to the latest version allowed. I wonder if the issue is caused by the fact that I am using a conda environment to be able to install `curvenote`. Would things be different if I instead installed node using the linux package manager. As in, is there a library that I do not have that causes this error?

**Comment by rowanc1** [comment]
Thanks for the report! :) We haven't heavily tested the watch/start on Linux yet, so that is likely the problem. @stevejpurves is firing things up at the moment.

Some quick tasks:
- [ ] remove the blocking watch (won't have live reload)
- [ ] figure out a standard way to watch for file system changes

I will probably cut a release soon to get the start command unblocked for you.

**Comment by stevejpurves** [comment]
I have just tested on ubuntu 18, installing node via `nvm` (my usual route, you might want to check it out anyways @johnjarmitage https://github.com/nvm-sh/nvm) and can reproduce your problem.

It looks like a deeper problem with `node` since `v14` but ne we think we can fix by changing what we use to watch files to `node-watch` rather than `fs-watch`

**Comment by stevejpurves** [comment]
see end of thread on: https://github.com/wclr/ts-node-dev/issues/143

**Comment by rowanc1** [comment]
This should hopefully be fixed in v0.4.12, which has just been released. @johnjarmitage the citations in your project should also be nicer after upgrading. :)

Closing, but feel free to post here if there are still issues!

**Comment by stevejpurves** [comment]
Just tested this on ubuntu 18 with node v14 & v16 - working well there 🤘🏼
